packages/core/bootstrap/src/lib/modules/logger.ts

```diff
@@ -2,8 +2,9 @@ import { uuid } from '../util'
 import pino from 'pino'
 import { wsRedactPaths } from '../middleware/ws/config'
 import { cloneDeep } from 'lodash'
+import { configRedactPaths } from '../config'
 
-export const paths = [...wsRedactPaths]
+const paths = [...wsRedactPaths, ...(configRedactPaths || [])]
```

Comment:
Had issues with config.test.ts throwing an error that configRedactPaths is not iterable. Unsure why it has problems with this static string[] and not with wsRedactPaths. Applied this check to avoid the issue but not a fan of having to do this. If anyone has a suggestion on what could be happening, that'd be super helpful.

Reply:
I do not fully understand what's causing this issue, but I found a better way to work around it.
It's failing because it's reading `configRedactPaths` from `index.ts`. If you create a new file, e.g. `config/logging.ts`, and place the array there, it will work.

I guess that's better than `(configRedactPaths || [])`? :man_shrugging:

Reply:
Interesting... I agree that's definitely better. Just updated it in the latest commit.